content/docs/04-clusters/03-edge/04-edgeforge-workflow/12-build-artifacts.md

```diff
@@ -0,0 +1,309 @@
+
+  </WarningBox>
+  <br />
+
+
+9. Add the following Kubernetes layer to your cluster profile. Select the K3s version 1.25.x because earlier in this how-to guide, you pushed a provider image compatible with K3s v1.25.2 to the ttl.sh image registry. 
+
+  |**Pack Type**|**Registry**|**Pack Name**|**Pack Version**| 
+  |---|---|---|---|
+  |Kubernetes|Public Repo|Palette Optimized K3s|`1.25.x`|
+
+
+10. Add the network layer to your cluster profile, and choose a Container Network Interface (CNI) pack that best fits your needs, such as Calico, Flannel, Cilium, or Custom CNI. For example, you can add the following network layer. This step completes the core infrastructure layers in the cluster profile.  
+
+  |**Pack Type**|**Registry**|**Pack Name**|**Pack Version**| 
+  |---|---|---|---|
+  |Network|Public Repo|Calico|`3.25.x`|
+    
+  
+11. Add add-on layers and manifests to your cluster profile per your requirements. 
+
+
+12. If there are no errors or compatibility issues, Palette displays the newly created complete cluster profile for review. Verify the layers you added, and finish creating the cluster profile. 
+
+
+13. The remaining steps will guide you to create an offline content bundle using the Palette Edge CLI and build the Edge installer ISO. 
+
+  To use Palette Edge CLI, you need a Spectro Cloud API key to authenticate and interact with the Palette API endpoint. To add a new API key, log in to Palette, click on the user **User Menu** at the top right, and select **My API Keys**, as displayed in the screenshot below. 
+
+  Fill in the required fields, such as the API key name and expiration date, and confirm your changes. Copy the key value to your clipboard to use in the next step.
+
+  ![Screenshot of generating an API key in Palette.](/tutorials/deploy-pack/registries-and-packs_deploy-pack_generate-api-key.png )
+
+  
+
+14. Set the API key by issuing the following command. Replace the `[USE-YOUR-API-KEY_HERE]` placeholder with your API key. The Palette Edge CLI will use this API key to authenticate itself when interacting with the Spectro Cloud API. Once authenticated, the Palette Edge CLI can access and manage resources in your Spectro Cloud account.
+<br />
+
+  ```bash
+  export API_KEY=[USE-YOUR-API-KEY_HERE]
+  ```
+
+
+15. Copy the cluster profile and project IDs from Palette to use in the next step. To locate these IDs, switch to [Palette](https://console.spectrocloud.com). Navigate to the left **Main Menu** and select **Profiles**. Click on the specific cluster profile to view its details. The cluster profile and project IDs are in the cluster details page URL. The cluster details page URL follows the `[Palette-URL]/projects/[PROJECT-ID]/profiles/cluster/[PROFILE-ID]` syntax. 
```

Comment:
Break these steps up. You are overloading several steps into one.  Also, let's show the user how to find the projectID the right way (see screenshot). The Cluster Profile ID is something I think the UX team is improving in the next release. 

![CleanShot 2023-07-24 at 15 30 01](https://github.com/spectrocloud/librarium/assets/29551334/e9acd2d7-3f41-46d9-8f88-145839b645c1)